src/dmd/init.d

```diff
@@ -251,6 +251,8 @@ struct DesigInit
 extern (C++) final class CInitializer : Initializer
 {
     DesigInits initializerList; /// initializer-list
+    Type type;              /// type that array will be used to initialize
+    bool sem;               /// true if semantic() is run
```

Comment:
This seems familiar, so I assume similar is used elsewhere. But wouldn't the semantic enum be more consistent?

Reply:
I copied it from ArrayInitializer